code/modules/jobs/job_types/den.dm

```diff
@@ -77,6 +77,125 @@ Settler
 		/obj/item/clothing/under/f13/machinist, \
 		/obj/item/clothing/under/f13/lumberjack, \
 		/obj/item/clothing/under/f13/roving)
+	r_pocket = pick(
+		/obj/item/flashlight/flare/torch, \
+		/obj/item/flashlight/flare)
+
+
+/*
+Preacher
+*/
+
+/datum/job/f13preacher
+	title = "Preacher"
+	flag = F13PREACHER
+	department_head = list("Captain")
+	department_flag = DEN
+	head_announce = list("Security")
+	faction = "Den"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "your deity."
+	selection_color = "#dddddd"
+
+	outfit = /datum/outfit/job/f13preacher
+
+	access = list()
+	minimal_access = list()
+
+
+
+/datum/job/f13preacher/after_spawn(mob/living/H, mob/M)
+	if(H.mind)
+		H.mind.isholy = TRUE
+
+	var/obj/item/storage/book/bible/booze/B = new
+
+	if(SSreligion.religion)
+		B.deity_name = SSreligion.deity
+		B.name = SSreligion.bible_name
+		B.icon_state = SSreligion.bible_icon_state
+		B.item_state = SSreligion.bible_item_state
+		to_chat(H, "There is already an established religion onboard the station. You are an acolyte of [SSreligion.deity]. Defer to the Chaplain.")
+		H.equip_to_slot_or_del(B, SLOT_IN_BACKPACK)
+		var/nrt = SSreligion.holy_weapon_type || /obj/item/nullrod
+		var/obj/item/nullrod/N = new nrt(H)
+		H.put_in_hands(N)
+		return
+
+	var/new_religion = "Christianity"
+	if(M.client && M.client.prefs.custom_names["religion"])
+		new_religion = M.client.prefs.custom_names["religion"]
+
+	var/new_deity = "Space Jesus"
+	if(M.client && M.client.prefs.custom_names["deity"])
+		new_deity = M.client.prefs.custom_names["deity"]
+
+	B.deity_name = new_deity
+
+
+	switch(lowertext(new_religion))
+		if("christianity")
+			B.name = pick("The Holy Bible","The Dead Sea Scrolls")
+		if("satanism")
+			B.name = "The Unholy Bible"
+		if("cthulhu")
+			B.name = "The Necronomicon"
+		if("islam")
+			B.name = "Quran"
+		if("scientology")
+			B.name = pick("The Biography of L. Ron Hubbard","Dianetics")
+		if("chaos")
+			B.name = "The Book of Lorgar"
+		if("imperium")
+			B.name = "Uplifting Primer"
+		if("toolboxia")
+			B.name = "Toolbox Manifesto"
+		if("homosexuality")
+			B.name = "Guys Gone Wild"
+		if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks", "meme", "memes")
+			B.name = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition")
+			H.adjustBrainLoss(100) // starts off retarded as fuck
```

Comment:
kek

Reply:
Pffft. That's all apparently in TG chaplain.